packages/nodejs/src/cli/diagnose.ts

```diff
@@ -0,0 +1,249 @@
+        data["host"]["running_in_container"]
+      )}`
+    )
+
+    this.print_newline()
+
+    console.log(`Configuration`)
+    console.log(
+      `  Environment: ${this.format_value(data["config"]["options"]["env"])}`
+    )
+
+    console.log(
+      `  debug: ${this.format_value(data["config"]["options"]["debug"])}`
+    )
+
+    console.log(`  log: ${this.format_value(data["config"]["options"]["log"])}`)
+
+    console.log(
+      `  endpoint: ${this.format_value(data["config"]["options"]["endpoint"])}`
+    )
+    console.log(
+      `  ca_file_path: ${this.format_value(
+        data["config"]["options"]["ca_file_path"]
+      )}`
+    )
+    console.log(
+      `  active: ${this.format_value(data["config"]["options"]["active"])}`
+    )
+    console.log(
+      `  push_api_key: ${this.format_value(
+        data["config"]["options"]["push_api_key"]
+      )}`
+    )
+
+    this.print_newline()
+
+    console.log(`Read more about how the diagnose config output is rendered`)
+    console.log(`https://docs.appsignal.com/nodejs/command-line/diagnose.html`)
+
+    this.print_newline()
+
+    console.log(`Validation`)
+    console.log(
+      `  Validating Push API key: ${this.colorize(
+        data["validation"]["push_api_key"]
+      )}`
+    )
+
+    this.print_newline()
+
+    console.log(`Paths`)
+
+    var contents = data["paths"]["appsignal.log"]["content"]
+
+    console.log(`  Current working directory`)
+    console.log(
+      `    Path: ${this.format_value(data["paths"]["working_dir"]["path"])}`
+    )
+
+    this.print_newline()
+
+    console.log(`  Log directory`)
+    console.log(
+      `    Path: ${this.format_value(data["paths"]["log_dir_path"]["path"])}`
+    )
+
+    this.print_newline()
+
+    console.log(`  AppSignal log`)
+    console.log(
+      `    Path: ${this.format_value(data["paths"]["appsignal.log"]["path"])}`
+    )
+    console.log(`    Contents \(last 9 lines\):`)
+    console.log(contents.slice(contents.length - 10).join("\n"))
+
+    console.log(`Diagnostics report`)
+    console.log(`  Do you want to send this diagnostics report to AppSignal?`)
+    console.log(`  If you share this report you will be given a link to`)
+    console.log(`  AppSignal.com to validate the report.`)
+    console.log(`  You can also contact us at support@appsignal.com`)
+    console.log(`  with your support token.`)
+
+    this.print_newline()
+
+    if (process.argv.includes("--no-send-report")) {
+      console.log(
+        `  Not sending report. (Specified with the --no-send-report option.)`
+      )
+    } else if (process.argv.includes("--send-report")) {
+      this.send_report(data)
+    } else {
+      const rl = readline.createInterface({
+        input: process.stdin,
+        output: process.stdout
+      })
+
+      let self = this
+      rl.question(
+        `  Send diagnostics report to AppSignal? (Y/n): `,
+        function (answer: String) {
+          switch (answer) {
+            case "":
```

Comment:
Not sure I like a non breaking switch statement. We can also make the default value "y" in the switch?

```suggestion
          switch (answer || "y") {
```


```js
> "" || "y"
'y'
```